app/frontend/src/views/admin/Submission.vue

```diff
@@ -102,7 +102,7 @@ export default {
     },
     operationTypeDisplayName(operationType){
       const obj =  this.operationTypes.find( ({ type })  => type === operationType);
-      return obj.display;
+      return (obj !== null) ? obj.display: 'not specified';
```

Comment:
Can just be a truthyness check, that would handle undefined as well as null
`if(obj)`

Reply:
In fact, this is still broken because it's undefined
https://silviculture-ipc-dev.pathfinder.gov.bc.ca/pr-111/#/admin/submission/9a471fde-bd0d-4a35-86f2-25a3a7e3eb3d
Gotta be careful with the JS null/undefined stuff

![image](https://user-images.githubusercontent.com/17445138/84446122-b6c82d80-abf9-11ea-8372-594aa449248b.png)

Reply:
ok. i thought obj would be 'undefined' if operationType (from the database) was equal to null.

Reply:
obj **is** undefined if find doesn't find anything (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find).
and then checking if it's null (rather than undefined) passes, and it tries to access a prop on undefined

Reply:
i thought ` x !== null`  means ' x is not eqaul to null and is not equal to 'undefined'
answer here: https://stackoverflow.com/questions/2647867/how-can-i-determine-if-a-variable-is-undefined-or-null

so you think I should put this?:
`      return (obj !== undefined) ? obj.display: 'not specified';`